boards/arm/nrf52810_pca10040/Kconfig.defconfig

```diff
@@ -0,0 +1,35 @@
+# Kconfig - nRF52810 PCA10040 board configuration
+#
+# Copyright (c) 2018 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_NRF52810_PCA10040
+
+config BOARD
+	default "nrf52810_pca10040"
+
+if GPIO_NRF5
+
+config GPIO_NRF5_P0
+	default y
+
+endif # GPIO_NRF5
+
+if UART_NRFX
```

Comment:
Is this needed @anangl ? Is 810 going to have only UARTE, instead?

Reply:
It is needed. It enables nrfx drivers for UARTs in general. The type of the driver (UART or UARTE) is selected with a different option.

Reply:
ok